### DISCUSSION TITLE
Future sample app?

### DISCUSSION THREAD

**Discussion by ThePalad1n** [pocketbase/pocketbase#129] [Q&A] [answered]
Will there be a future sample app separate from this repo?

**Comment by ganigeorgiev** [comment, marked as answer]
What do you mean by a sample app?

There is already a minimal example, that is used for the prebuilt binaries - https://github.com/pocketbase/pocketbase/blob/master/examples/base/main.go

Or you are looking for something more advanced?

**Reply by ThePalad1n** [reply]
Something more advanced. I understand this is brand new and was just curious. I am sure you have a lot going on with all of this but congrats on all the new incoming support.
